python/servo/build_commands.py

```diff
@@ -25,8 +24,14 @@
 from servo.command_base import CommandBase, cd
 
 
-def is_headless_build():
```

Comment:
This function shouldn't be removed, we still need `$SERVO_HEADLESS` to work.

We should build headless if either `--headless` is passed or if `$SERVO_HEADLESS` is defined.